CONTRIBUTING.md

```diff
@@ -20,16 +20,18 @@ Please contribute links to packages/projects you have used or are familiar with.
 
 ## Quality standard
 
-To stay on the list, package repositories should adhere to these quality standards:
+To be on the list, project repositories should adhere to these quality standards:
 
-- Generally useful to the community
-- Functional
-- Actively maintained (even if that just means acknowledging open issues when they arise)
-- Stable, or progressing toward stable
-- Documented (preferably godoc.org)
-- Tests are preferred (when possible)
+- Code is functional
+- Generally useful to the wider community of Go programmers
+- Actively maintained
+  - Regular, recent commits
+  - Or, for finished projects, issues and pull requests are responded to
+- Stable or progressing toward stable
+- Thoroughly documented (README, godoc comments, etc.)
+- Tests, where practical
```

Comment:
Using goreportcard.com sounds fair. It includes lint, vet and a few others in one, which is great.

However, I see there's a yellow notice on top that says "Go Report Card shutting down". http://goreportcard.com/shutdown/